news/2018-06-30-cwc-2018-recap-quarter-finals.md

```diff
@@ -0,0 +1,51 @@
+date: 2018-06-30 01:00:00 +0000
+---
+
+Another week of historic fruit catching action down, just three more to go. This year's osu!catch World Cup Quarter Finals bracket was potentially more hyped than any before. Read to find out who booked an early trip to Finals Week!
+
+![](/wiki/shared/news/banners/CWC_2018_banner.jpg)
+
+Hello everyone and welcome! With regular recapper [JBHyperion](https://osu.ppy.sh/u/4879508) on vacation, I, [Dohland](https://osu.ppy.sh/u/5220511) present to you the osu!catch World Cup 2018 Quarter Finals Recap! We had high expectations going into the weekend with one of the most stacked Winner’s Bracket Quarterfinals we’ve ever seen. Let’s see who came out on top!
+
+<iframe src="https://player.twitch.tv/?autoplay=false&video=v277697607" frameborder="0" allowfullscreen="true" scrolling="no" height="378" width="100%"></iframe>
+
+## Quarter Finals
+
+We kicked the weekend off with a banger! Germany, who had an impressive 5-0 win over the Netherlands last weekend, battled against South Korea, who took down Japan in the Round of 16. South Korea got things rolling with Hidden, while Germany elected to go with Hard Rock.
+
+Although these mod selections surprised no one, the outcome sure did – Korea took an early 2-0 lead after breaking the Germans' pick. Germany wouldn't go down easily though, as they fought back to level the match at 3-3. Their counter-attack was ultimately to no avail however, as Korea continued winning in the same way they had last week. Despite giving up some early points, they would win on Double Time and No Mod to take a 5-3 victory.
+
+Next up was another Winner's Bracket encounter between Indonesia and Hong Kong. Both teams started off strong by winning their opening picks, but things would take a turn for the worse for Hong Kong, who dropped their next two picks and found themselves staring at a 4-1 deficit. After two more break points we would arrive at our final scoreline of 5-2. Indonesia took the win and advance to Finals Week for a third year in a row!
+
+Unfortunately, we saw our first two “win by default” results of the tournament this weekend. Sweden and Finland forfeited their Losers' Bracket matches to Poland and Mexico respectively. This eliminated the two Scandinavian teams from the tournament. Hopefully we will see them again next year.
+
+Next up, Netherlands clashed with Japan – both teams were at risk of being eliminated and sent home, so they put their all on the line. Netherlands bounced back strongly from last week’s loss, taking an early lead and never looking back. Japan was unable to maintain consistency with the Dutch team and would eventually fall 5-1.
+
+Then came the match everyone was waiting for. China. The United States. So much at stake for both teams. After all the amazing moments and matches we've had so far in this tournament, it was hard to imagine anything topping what we had already seen. But leave it to the top two teams to shock us all!
```

Comment:
> for. China. The United States.

Don't know if this formatting is right.

Reply:
From a speaking pov I understand what Dohland was going for here, but in text format it does look a bit strange. Made a more fluid sentence structure.